packages/function-tree-firebase-admin/src/TestTasks.js

```diff
@@ -0,0 +1,104 @@
+import FunctionTree from 'function-tree'
+import originalWebsocket from 'faye-websocket'
+import Provider from './Provider'
+import proxyquire from 'proxyquire'
+import FirebaseServer from 'firebase-server'
+import FirebaseProvider from './Provider'
+
+const firebase = proxyquire('firebase', {
+	'faye-websocket': {
```

Comment:
It was fixed? There is something really weird happening there... I could not remove the tabs. Tried everything, even a tool to remove tabs. So I manually used spaces... but now they are tabs again? But all checks has passed?